arxiv/browse/util/author_affil.py

```diff
@@ -0,0 +1,363 @@
+"""Parse Authors lines to extract author and affiliation data."""
```

Comment:
Per Slack discussion 26 November and comments in ARXIVNG-1097: author name parsing should be added to ``arxiv.util``. ``arxiv.util.author_affil`` would be ok.